### PR TITLE
created catalog using grid

### PR DESCRIPTION
**Pull request by vladyslav-krainyk**
- [DEMO LINK](https://vladyslav-krainyk.github.io/layout_catalog_grid/)

### CODE REVIEW

**Review by stanislav-topikha — CHANGES_REQUESTED** [review]
you dont need all those changes, modify only one file:` _catalog.scss`

File: src/styles/_catalog.scss [hunk 1]
```diff
@@ -1,22 +1,31 @@
+$card-width: 200px;
+$scrin-size:
```
Comment (ivanchuk-yurii) [inline src/styles/_catalog.scss:2]:
```suggestion
$screen-size:
```
typo

File: src/styles/blocks/card.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.card {
```
Comment (ivanchuk-yurii) [inline src/styles/blocks/card.scss:1]: remove this

File: src/styles/blocks/catalog.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
+.catalog {
```
Comment (ivanchuk-yurii) [inline src/styles/blocks/catalog.scss:1]: and this

File: src/styles/utils/_variables.scss [hunk 4]
```diff
@@ -0,0 +1,5 @@
+$card-width: 200px;
```
Comment (ivanchuk-yurii) [inline src/styles/utils/_variables.scss:1]: this too

File: src/styles/_catalog.scss [hunk 5]
```diff
@@ -1,22 +1,31 @@
...
+      }
+    }
+  }
 
   &__card {
     display: flex;
     justify-content: center;
     align-items: center;
-    width: 200px;
+    width: $card-width;
```
Comment (ivanchuk-yurii) [inline src/styles/_catalog.scss:26]: you don't need this since you've set width in `grid-template-columns`

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]

**Pull request closed without merging**